Tasks/VsTest/vstest.ts

```diff
@@ -36,6 +36,7 @@ try {
     var runInParallel: boolean = tl.getBoolInput('runInParallel');
     var tiaEnabled = tl.getVariable('tia.enabled');
     var fileLevel = tl.getVariable('tia.filelevel');
+    var tiaRebaseLimit = tl.getVariable('tia.rebaselimit');
```

Comment:
Will take this in another PR, since there are some open questions for UI. Pushing in the remaining for now.